database/migrations/default/1754674089763_inj_sev_reason_readable/up.sql

```diff
@@ -0,0 +1,106 @@
+--
+-- Adds human-friendly values to patient_injry_sev_reason
+--
+CREATE OR REPLACE FUNCTION public.update_ems_patient_injry_sev()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+BEGIN
+    case
+    --
+    -- fatal injuries
+    --
+    when (lower(NEW.pcr_provider_impression_primary) = 'death on scene') then
+        NEW.patient_injry_sev_id = 4;
+        NEW.patient_injry_sev_reason = 'Reported as "death on scene"';
```

Comment:
The updates to this function are in each line that assigns `NEW.patient_injry_sev_reason` 👍